test/jit/test_backends.py

```diff
@@ -685,3 +687,75 @@ def test_execution(self):
     @skipIfRocm
     def test_errors(self):
         self.error_module_compiler_test.test_errors()
+
+"""
+Unit Tests for Nnapi backend with delegate
+"""
+# This is needed for IS_WINDOWS or IS_MACOS to skip the tests.
+@unittest.skipIf(TEST_WITH_ROCM or IS_SANDCASTLE or IS_WINDOWS or IS_MACOS or IS_FBCODE,
+                 "Non-portable load_library call used in test")
+class NnapiBackendPReLUTest(JitTestCase):
+    """
+    Test lowering a simple PRelU module to Nnapi backend.
+    TODO: After Nnapi delegate is finished T91991928,
+    add tests for running the module (currently only lowers the model)
+    """
+    def setUp(self):
+        super().setUp()
+        # Load nnapi delegate library
+        torch_root = Path(__file__).resolve().parent.parent.parent
+        p = torch_root / 'build' / 'lib' / 'libnnapi_backend.so'
+        torch.ops.load_library(str(p))
+
+        # Nnapi set up
+        # Avoid saturation in fbgemm
+        torch.backends.quantized.engine = 'qnnpack'
+
+        libneuralnetworks_path = os.environ.get("LIBNEURALNETWORKS_PATH")
```

Comment:
Just curious: why is this needed?

And should it fail if it can't?

Reply:
Perhaps, @axitkhurana or @dreiss can provide more insight. I've tried only converting to the NNAPI backend without these lines. It seemed run successfully, but I believe they are more necessary for execution, especially the loading of libneuralnetworks, if you're also referring to that.

Reply:
Yes, we use it to test nnapi execution on CPU and compare the results with pytorch, conversion should work without it